lib/Runtime/Library/JavascriptError.cpp

```diff
@@ -159,6 +159,54 @@ namespace Js
     }
 #endif
 
+    Var JavascriptAggregateError::NewAggregateErrorInstance(RecyclableObject* function, CallInfo callInfo, ...)
+    {
+        PROBE_STACK(function->GetScriptContext(), Js::Constants::MinStackDefault);
+        ARGUMENTS(args, callInfo);
+        ScriptContext* scriptContext = function->GetScriptContext();
+        JavascriptLibrary* library = scriptContext->GetLibrary();
+        JavascriptAggregateError* pError = scriptContext->GetLibrary()->CreateAggregateError();
+        Var newTarget = args.GetNewTarget();
+        RecyclableObject* errors = args.Info.Count > 1 ? JavascriptOperators::IterableToList(VarTo<RecyclableObject>(args[1]), scriptContext) : library->CreateArray(0);
+        Var message = args.Info.Count > 2 ? args[2] : library->GetUndefined();
+        pError->SetErrors(errors);
+
+        return JavascriptError::NewInstance(function, pError, callInfo, newTarget, message);
+    }
+
+    Var JavascriptAggregateError::EntryGetterErrors(RecyclableObject* function, CallInfo callInfo, ...)
+    {
+        PROBE_STACK(function->GetScriptContext(), Js::Constants::MinStackDefault);
+
+        ARGUMENTS(args, callInfo);
+        ScriptContext* scriptContext = function->GetScriptContext();
+
+        AssertMsg(args.Info.Count > 0, "Should always have implicit 'this'");
+
+        Assert(!(callInfo.Flags & CallFlags_New));
+
+        if (args[0] == 0 || !JavascriptOperators::IsObject(args[0]))
+        {
+            JavascriptError::ThrowTypeError(scriptContext, JSERR_This_NeedObject, _u("AggregateError.prototype.errors"));
+        }
+
+        if (!VarIs<JavascriptAggregateError>(args[0])) {
+            JavascriptError::ThrowTypeError(scriptContext, JSERR_This_NeedObject, _u("AggregateError.prototype.errors"));
+        }
+
+        JavascriptLibrary* library = scriptContext->GetLibrary();
+        JavascriptAggregateError* thisError = VarTo<JavascriptAggregateError>(args[0]);
+
+        RecyclableObject* iterator = JavascriptOperators::GetIterator(thisError->GetErrors(), scriptContext);
```

Comment:
Should do a nullptr check on the errors property - and return a 0 length array if errors is nullptr.